dojo/api_v2/serializers.py

```diff
@@ -1735,16 +1739,15 @@ def save(self, push_to_jira=False):
                                                 commit_hash=commit_hash, push_to_jira=push_to_jira,
                                                 close_old_findings=close_old_findings,
                                                 group_by=group_by, api_scan_configuration=api_scan_configuration,
-                                                service=service, do_not_reactivate=do_not_reactivate,
-                                                create_finding_groups_for_all_findings=create_finding_groups_for_all_findings)
```

Comment:
Was this intentional?

Reply:
@Maffooch no, thanks for pointing out. Something went wrong when sync with the PR #7172